src/index.js

```diff
@@ -74,6 +77,8 @@ const VueHtmlToPaper = {
         win.document.close();
         win.focus();
         win.print();
+        autoClose && win.document.close();
+        autoClose && win.close();
         setTimeout(function () {window.close();}, 1);
```

Comment:
I think that the author remove that option in favor of always close the window with this line. But actually, this isn't working either 😅  In order for this to work, it has to be `win.close` inside the setTimeout.